emscripten.py

```diff
@@ -664,7 +664,8 @@ def align_memory(addr):
 
 
 def align_static_bump(metadata):
-  metadata['staticBump'] = align_memory(metadata['staticBump'])
+  if not shared.Settings.RELOCATABLE:
```

Comment:
why? (is this only for fastcomp?)

Reply:
You are right this looks odd.   The reason is that the static bump gets written into the shared library as the static data size, which is separate from its alignment.

Let me see if I can refactor to make this look less odd.

Reply:
This is not part of the change anymore